github/repos.go

```diff
@@ -245,6 +246,7 @@ func (s *RepositoriesService) ListAll(ctx context.Context, opt *RepositoryListAl
 	if err != nil {
 		return nil, nil, err
 	}
+	req.Header.Set("Accept", mediaTypeCodesOfConductPreview)
```

Comment:
This line should have a blank line above it, and a `// TODO: remove custom Accept header when license support fully launches` comment for consistency.